domains/ehab.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ehabmansour1",
+        "email": "mansehab990@gmail.com"
+    },
+    "record": {
+        "CNAME": "ehabmansour1.github.io"
```

Comment:
I cannot reach your website, redirects me to `https://ehabmansour.tk/` which returns
> This site can’t be reachedCheck if there is a typo in ehabmansour.tk.
> DNS_PROBE_FINISHED_NXDOMAIN

Reply:
https://ehabmansour1.github.io/my_portfolio/

Reply:
now i can access it